CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 11.3.13 (06/09/23)
+
+* Server Access
+  * Fixed issue with proxy port not being included in SSH config generated by `tsh config`. [#27547](https://github.com/gravitational/teleport/pull/27547)
+* Access
+  * Added support for locking any Teleport service with `tctl lock` command. [#27444](https://github.com/gravitational/teleport/pull/27444)
+* Machine ID
+  * Added Prometheus endpoint to tbot. [#27434](https://github.com/gravitational/teleport/pull/27434)
+* SAML IdP
+  * Fixed issue with SAML IdP advertising incorrect port in some cases. [#27378](https://github.com/gravitational/teleport/pull/27378)
```

Comment:
nit: The fix for this was pulled in because it could impact more than just the SAML IdP, but the SAML IdP isn't in v11.